include/boost/beast/core/detail/bind_default_executor.hpp

```diff
@@ -67,38 +67,49 @@ class bind_default_executor_wrapper
             h_, this->get());
     }
 
+    // The invocation hook is no longer defined because it customises behaviour
+    // without forwarding to a user's hook.
+
+#if 0
```

Comment:
Should we just delete the code and comment?

Reply:
sure, why not?